### PR TITLE
[RHOAIENG-5073] Add pod port fowarding for raw due to headless service

### PR DESCRIPTION
**Pull request by tarukumar**

### CODE REVIEW

**Review by mwaykole — APPROVED** [review]

File: ods_ci/tests/Resources/CLI/ModelServing/llm.resource [hunk 1]
```diff
@@ -388,7 +388,8 @@ Compile Deploy And Query LLM model
     ...    namespace=${namespace}
     Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${isvc_name}
     ...    namespace=${namespace}
-    IF    ${IS_KSERVE_RAW}     Start Port-forwarding    namespace=${namespace}    model_name=${model_name}
+    ${pod_name}=  Get Pod Name    namespace=${namespace}    label_selector=serving.kserve.io/inferenceservice=${isvc_name}
```
Comment (bdattoma) [inline ods_ci/tests/Resources/CLI/ModelServing/llm.resource:391]: what if we have multiple replicas? we should take only first pod no?

Comment (bdattoma) [reply ods_ci/tests/Resources/CLI/ModelServing/llm.resource:391]: talked offline

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_tgis.robot [hunk 2]
```diff
@@ -85,16 +86,16 @@ Verify User Can Deploy Multiple Models In The Same Namespace
     ...    namespace=${test_namespace}
     Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${model_one_name}
     ...    namespace=${test_namespace}
-    Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${model_two_name}
-    ...    namespace=${test_namespace}
```
Comment (lugi0) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_tgis.robot:89]: why remove this wait?

**Review by lugi0 — APPROVED** [review]
one question but lgtm otherwise

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_tgis.robot (hunk 2 above)
Comment (tarukumar) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_tgis.robot:89]: mistake..i will fix it

Comment (tarukumar) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_tgis.robot]: done

**Review by mwaykole — APPROVED** [review]

**Review by lugi0 — APPROVED** [review]

**Review by bdattoma — APPROVED** [review]

**Pull request merged**